src/uct/ib/dc/dc_mlx5.c

```diff
@@ -40,7 +40,7 @@ ucs_config_field_t uct_dc_mlx5_iface_config_sub_table[] = {
      ucs_offsetof(uct_dc_mlx5_iface_config_t, super),
      UCS_CONFIG_TYPE_TABLE(uct_rc_iface_common_config_table)},
 
-    {"RC_", "", NULL,
+    {"RC_", "SRQ_TOPO=list", NULL,
```

Comment:
maybe add the comment why we prefer `list` WQEs representation (e.g. performance reason or etc)?